python-spec/src/somacore/base.py

```diff
@@ -10,6 +10,7 @@
 from typing_extensions import LiteralString, Self
 
 from . import options
+from .types import ContextBase
```

Comment:
minor nit: in somacore import modules, not things inside modules, so `from . import types` and use `types.ContextBase` below